great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -0,0 +1,17 @@
+from great_expectations.core.config_peer import ConfigPeer
+from great_expectations.data_context.types.base import BaseYamlConfig
+
+
+class AbstractDataContext(ConfigPeer):
+    """
+    Base class for all DataContexts that contain all context-agnostic data context operations.
+
+    The class encapsulates most store / core components and convenience methods used to access them, meaning the
+    majority of DataContext functionality lives here.
+
+    TODO: eventually the dependency on ConfigPeer will be removed and this will become a pure ABC.
+    """
+
+    @property
+    def config(self) -> BaseYamlConfig:
```

Comment:
@Shinnnyshinshin Maybe simply call super().config -- since it extends the `ConfigPeer` for now?

Reply:
oo great point :) thank you

Reply:
can be removed